lib/orderbook/OrderBook.ts

```diff
@@ -315,15 +315,19 @@ class OrderBook extends EventEmitter {
   /**
    * Send all local orders to a given peer in an [[OrdersPacket].
    * @param reqId the request id of a [[GetOrdersPacket]] packet that this method is responding to
+   * @param pairIds comes from Handshakestate, to send only supported pairs to peer
```

Comment:
it actually doesn't come from `handshakeState`, but from the request packet payload, which we shouldn't mention here (unless we'll rather emit the packet entirely from `Pool`). `handshakeState` supposed to be used for the unilateral `broadcastOrder`.